package.json

```diff
@@ -1,56 +1,65 @@
 {
   "name": "nteract.io",
-  "version": "1.0.0",
   "description": "The server for nteract.io",
-  "scripts": {
-    "dev": "next",
-    "build": "next build",
-    "flow": "flow",
-    "start": "next start",
-    "prettier": "prettier --write './**/*.{js,scss}'"
-  },
-  "lint-staged": {
-    "{lib,spec}/**/*.{js,scss}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nteract/nteract.io.git"
-  },
+  "version": "1.0.0",
   "author": "nteract contributors",
-  "license": "BSD-3-Clause",
   "bugs": {
     "url": "https://github.com/nteract/nteract.io/issues"
   },
-  "homepage": "https://github.com/nteract/nteract.io#readme",
   "dependencies": {
     "autoprefixer": "^8.4.1",
     "babel-plugin-module-resolver": "^3.1.1",
     "babel-plugin-wrap-in-js": "^1.1.1",
+    "express": "^4.16.3",
     "fetch-github-organization": "^1.0.3",
     "glob": "^7.1.2",
     "in-view": "^0.6.1",
     "isomorphic-fetch": "^2.2.1",
     "mdi": "^2.2.43",
-    "next": "^6.0.0",
+    "mdi-react": "^3.3.0",
+    "next": "^7.0.2",
     "node-sass": "^4.9.0",
     "normalize-scss": "^7.0.1",
     "nteract-members": "^1.0.8",
+    "polished": "^1.9.2",
     "postcss-easy-import": "^3.0.0",
     "postcss-loader": "^2.1.4",
     "raw-loader": "^0.5.1",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-fns": "^1.4.0",
     "react-headroom": "^2.2.2",
+    "react-spring": "^5.1.7",
     "react-syntax-highlighter": "^7.0.2",
-    "sass-loader": "^7.0.1"
+    "sass-loader": "^7.0.1",
+    "styled-components": "^3.2.6",
```

Comment:
Do we want to update to v4 (just released) https://www.styled-components.com/releases#v4.0.0 while we are refactoring?

Reply:
Great! I also should get rid of the sass-loader.